### PR TITLE
[FIX] cpu loss in actions

### PR DESCRIPTION
**Pull request by tkornuta-nvidia**
This PR introduces a fix in training function when backpropagating on CPU

### CODE REVIEW

File: nemo/backends/pytorch/actions.py [hunk 1]
```diff
@@ -1334,7 +1334,11 @@ def train(
                         final_loss.backward(bps_scale.to(final_loss.get_device()))
                     # single device (CPU or GPU)
                     else:
-                        final_loss.backward(bps_scale.to(final_loss.get_device()))
+                        # Fix (workaround?) enabling to backpropagate gradiens on CPUs.
+                        if final_loss.get_device() < 0:
+                            final_loss.backward(bps_scale.to(final_loss))
```
Comment (okuchaiev) [inline nemo/backends/pytorch/actions.py]: this should looks like a bug.
See https://pytorch.org/docs/stable/tensors.html#torch.Tensor.to
should be just "final_loss.backaward(bps_scale) in this case

Comment (tkornuta-nvidia) [reply nemo/backends/pytorch/actions.py]: Not sure whether I understand you... are you saying this is PT bug? let's meet and discuss that, maybe it will be easier...

Still, without that workaround NeMo isn't working on CPUs at all...

Comment (okuchaiev) [reply nemo/backends/pytorch/actions.py]: like what does it mean tensor.to(**tensor**)

Comment (tkornuta-nvidia) [reply nemo/backends/pytorch/actions.py]: You are right, so it should be simply:
final_loss.backward(bps_scale) ??

Comment (okuchaiev) [reply nemo/backends/pytorch/actions.py]: yes

**Comment by tkornuta-nvidia** [comment]
@blisc will update changelog in the next PR (when will finish NM configuration - stage 2 ;) )
